src/test/suite/connectionManager.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Connection, ConnectionManager, Context, guid } from '../../connectionManager';
+
+export function connectionManagerTester(): void {
```

Comment:
As time goes on we'll need to have a proper testing framework but this will work for now!